docker/Gemfile

```diff
@@ -1,5 +1,6 @@
 source 'https://rubygems.org'
 
 gem 'rubocop', '~>0.49'
+gem 'goodcheck', '~>0.3'
```

Comment:
Should this constraint be moved higher? Looks like 1.4.0 has been released.

Reply:
Looks like they've been busy. I don't see anything that looks like it would be breaking in their changelog, but I'll update my lint-review deployment to see how it behaves.